shared/common-adapters/icon.js.flow

```diff
@@ -2,332 +2,945 @@
 
 import React, {Component} from 'react'
 
-export type IconType = 'GPG-export'
```

Comment:
all old icons were deleted and new icons re-exported / shrunken. some renamed to make things more normalized